src/scripts/interfaces/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+The Licensed Work is (c) 2023 Sygma
+SPDX-License-Identifier: LGPL-3.0-only
+*/
+import { Transfer } from "@prisma/client"
+
+export interface IFixInterface {
+  executeAction: (transfer: Transfer) => void
```

Comment:
Maybe rename it to something like `executeTransferAction` because we might need to fix some other database entries also (eg. Fees, Deposits, Executions...)